src/index.jsx

```diff
@@ -63,7 +65,7 @@ class CircularColor extends PureComponent {
 
   handleClick = (event) => {
     const { size } = this.props;
-    const { innerRadius, outerRadius } = this.statics;
+    const { innerRadius, outerRadius, epsilon } = this.statics;
```

Comment:
I guess epsilon shouldn't be a class member. I'd recommend specifying it as a static property.

Reply:
Or it can be just a global variable